Samples/OLED1Xplained/OLED1.lpr

```diff
@@ -0,0 +1,152 @@
+program OLED1;
```

Comment:
Will not merge this example in this form as it is not following mbf concepts

Reply:
See above.
That is ok for me.
Its just a proof of concept.